unison-src/transcripts/fix1356.output.md

```diff
@@ -85,10 +85,6 @@ Let's check that we see both docs:
 ```ucm
 .> docs x
 
-  1. x.doc       : Doc
-  2. #v8f1hhvs57 : Doc
-  
-  Tip: Try using `display 1` to display the first result or
-       `view 1` to view its source.
+  I am the documentation for x, and I now look better
```

Comment:
This change is just due to it looking in the file for docs, not just the codebase.